File: 58-LengthOfLastWord/58-LengthOfLastWord.js
// Last updated: 24/04/2026, 10:05:46
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let lastword = s.trim( ).split(" ").pop();
    let count =0;
    for(var i =0; i<=lastword.length-1; i++){
        count++;
    }
    return count
};

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 24/04/2026, 10:05:43
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;

        for (int i = s.length() - 1; i >= 0; i--) {
            if (s[i] == ' ' && count > 0){
               break;
            } 
            if(s[i] != ' '){
                count++;
            }
        }

        return count;
    }
};